src/content/docs/en/reference/integrations-reference.mdx

```diff
@@ -396,6 +406,30 @@ The **`stage`** denotes how this script (the `content`) should be inserted. Some
 
 A read-only copy of the user-supplied [Astro config](/en/reference/configuration-reference/). This is resolved _after_ other integrations have run.
 
+#### `setAdapter` option
```

Comment:
Just checking that this isn't new (4.14) but was previously undocumented, so it doesn't need a `<Since />`?

Reply:
Yes exactly. It was only mentioned in the quick reference and thought it wouldn't hurt to add it